fpx-lib/src/data/util.rs

```diff
@@ -63,6 +63,18 @@ where
     }
 }
 
+impl<T> Serialize for Json<T>
```

Comment:
this can be derived no?

Reply:
That is actually a good point. However thinking about it, since we are using a custom deserializer that only accepts a json encoded string, we probably also need a serializer that only outputs json encoded string. :thinking: